docs/guides/getting-started.md

```diff
@@ -0,0 +1,95 @@
+---
+page_title: Getting Started
+---
+
+# Getting Started
+
+## Installing Terraform
+
+See [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) for more information.
+
+## Declaring a Task
+
+In an empty directory:
+
+1. Create a directory named `shared` to store input data and output artifacts.
+2. Create a file named `main.tf` with the following contents:
+
+```hcl
+terraform {
+  required_providers {
+    iterative = {
+      source  = "iterative/iterative"
+    }
+  }
+}
+
+provider "iterative" {}
+
+resource "iterative_task" "example" {
+  name  = "example"
+  cloud = "aws" # or any of: gcp, az, k8s
+
+  directory = "${path.root}/shared"
+
+  script = <<-END
+    #!/bin/bash
+    echo "Hello World!" > greeting.txt
+  END
+}
+```
+
+## Initializing Terraform
+
+```console
+$ terraform init
+```
+
+This command will:
+
+1. Download and install the Iterative Provider.
+2. Initialize Terraform in the current directory.
+
+~> **Note** None of the subsequent commands will work without first setting some [authentication environment variables](https://registry.terraform.io/providers/iterative/iterative/latest/docs#authentication)
```

Comment:
Copy edit
```suggestion
> **Note**: None of the subsequent commands will work without first setting [authentication environment variables](https://registry.terraform.io/providers/iterative/iterative/latest/docs#authentication).
```

Reply:
Is there any good reason for removing callouts in favor of block quotes? 🤔 
https://www.terraform.io/docs/registry/providers/docs.html#callouts

Reply:
I think `~>` and `->` render exactly like that right? Whereas `>` turns into a block quote (appropriate for notes IMO). It's what we usually do in product docs but this is stylistic so completely up to you.

Reply:
It's an special syntax for the provider documentation. You can preview how it would look here: https://registry.terraform.io/tools/doc-preview

Reply:
Ah very cool! I had no idea. Then no need to use block quotes but there are also small copy edits in some of my suggestions. Thanks

Reply:
```suggestion
~> **Note:** None of the subsequent commands will work without first setting some [authentication environment variables](https://registry.terraform.io/providers/iterative/iterative/latest/docs#authentication).
```